calcom/cal.com
# fix: seat enabled booking and rescheduling

## What does this PR do?

This change Fixes #12188

## Requirement/Documentation

- If there is a requirement document, please, share it here.

## Type of change
![1](https://github.com/calcom/cal.com/assets/90861780/21059c89-f518-416f-aca3-543c05f51559)
![2](https://github.com/calcom/cal.com/assets/90861780/50d3d2d4-5ed0-41c0-8d41-23ca9d98f104)
![3](https://github.com/calcom/cal.com/assets/90861780/0f7ab366-2503-408b-a7c5-fbefc6ed911e)
![4](https://github.com/calcom/cal.com/assets/90861780/bf81bbdb-6f9f-4d3f-9c84-b8e70aa41ca5)
![5](https://github.com/calcom/cal.com/assets/90861780/6dbd37f1-1163-4aba-a154-27d30970f1a9)
![6](https://github.com/calcom/cal.com/assets/90861780/4dc65345-b962-4555-8ab1-a9012d52e5db)





## How should this be tested?



As mentioned on the issue booking and rescheduling does not add new booking after one user booked at the same time and date

- Are there environment variables that should be set?
- What are the minimal test data to have?
- What is expected (happy path) to have (input and output)?
- Any other important info that could help to test that PR

## Mandatory Tasks


## Checklist
